lib/nostrum/api.ex

```diff
@@ -99,22 +99,23 @@ defmodule Nostrum.Api do
     - `pid` - Pid of the shard.
     - `status` - Status of the bot.
     - `game` - The 'playing' text of the bot. Empty will clear.
+    - `type` - The type of status to show.
     - `streaming` - URL of twitch.tv stream
   """
   @spec update_status(pid, status, String.t) :: :ok
```

Comment:
This spec is not correct (was incorrect prior to your changes) as it's missing the default parameters, could it be fixed to be in line with what's expected?